sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1594,6 +1450,168 @@ problem this extension currently aims to solve, it is the responsibility of the
 user to decide the device each command will be processed for, not the SYCL
 runtime.
 
+=== Storage Lifetimes [[storage-lifetimes]]
```

Comment:
We've added the `assume_data_outlives_buffer` property as suggested, and also added a similar `assume_buffer_outlives_graph` property because we also haven't implemented extending buffer lifetimes yet, so users can work around this to avoid an error as well.